src/lbaf/Utils/logger.py

```diff
@@ -66,7 +66,7 @@ def msgcolor(msg):
 
 def logger(
         name: str = "root",
-        level: Union[str, None] = None,
+        level: Union[str, None] = 'info',
```

Comment:
Please use double quotes for strings that are longer than a single character (this is one of the unwritten style rules of LBAF)